narwhal/worker/src/processor.rs

```diff
@@ -44,6 +44,8 @@ impl Processor {
 
                         // Store the batch.
                         store.write(digest, batch).await;
+                        // Wait until the batch was written
+                        let _ = store.notify_read(digest).await;
```

Comment:
Are we planning to deprecate this processor completely?

Reply:
Yes, the latest commits remove it completely.